### ISSUE TITLE
Can't able to access Tasklist and Operate, although ingress is used

### ISSUE THREAD

**Issue by RViththakan** [camunda/camunda-platform-helm#418]
Hi All,

I have deployed Camunda 8.0.13. I have tried to expose Tasklist and Operate with the NodePort and Ingress. When exposing it through Ingress and NodePort it is always throwing the same error

http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/auth?client_id=tasklist

Is there anyways to expose the tasklist and operate with identity.keycloak.auth enabled true?

Thanks in advance

**Comment by aabouzaid** [comment]
Hi @RViththakan, please give steps to reproduce the issue, the values file used for the deployment, and the exact error you see.

**Comment by RViththakan** [comment]
> Hi @RViththakan, please give steps to reproduce the issue, the values file used for the deployment, and the exact error you see.

Hi @aabouzaid,
I deployed Camunda, Followed the general installation steps only.
I need to expose tasklist and operate through the web browser. 

When deploying the Camunda, 
```
identity:
   enabled: false
   keycloak:
       fullname: ""
   auth:
       enabled: false
```
the auth was disabled.

All the pods are running except the optimize. When describing the optimize pod, there is an exception like
Error: secret "camunda-optimize-identity-secret" not found.

When disabling the identity.keycloak.auth the operate and tasklist can be exposed from NodePort. The only drawback was the optimize pod is not in running state, only in CreateContainerConfigError. 

After enabling the auth,
 ```
identity:
   enabled: false
   keycloak:
       fullname: ""
   auth:
       enabled: true
```
and deployed Camunda again,

All the pods are running. But when trying to expose the tasklist or operate through the NodePort, the following error riseup;
`http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/auth?client_id=tasklist`

I have tried the following solution also. But that didn't work.
[Link for the solution](https://forum.camunda.io/t/keycloak-redirection-url-for-camunda-identity/37517/3)

Is there any way to deploy Camunda enabling auth and expose the service tasklist and operate through browser?
